lib/yabeda/dsl/class_methods.rb

```diff
@@ -38,6 +38,16 @@ def group(group_name)
         @group = nil
       end
 
+      def adapter(adapter_name)
+        @adapter_name = adapter_name
+        yield if block_given?
```

Comment:
Love that you thought about usage with groups and very curious about this `yield`. How you planned to use it?

It may worth to be cherry-picked into separate pull request!

Reply:
I did it for this:
   ```rb
   adapter :prometheus do
     include_group :sidekiq
   end
   ```
Initialize the group configuration, before or after that declare that this is a group of only this adapter: `sidekiq` group only for `prometheus` adapter.